app/default_overrides.go

```diff
@@ -92,6 +117,24 @@ func (crisisModule) DefaultGenesis(cdc codec.JSONCodec) json.RawMessage {
 	})
 }
 
+type ibcModule struct {
+	ibc.AppModuleBasic
+}
+
+// DefaultGenesis returns custom x/crisis module genesis state.
+func (ibcModule) DefaultGenesis(cdc codec.JSONCodec) json.RawMessage {
+	// per ibc documentation, this value should be 3-5 times the expected block
+	// time. The expected block time is 15 seconds, therefore this value is 75
+	// seconds.
+	maxBlockTime := appconsts.GoalBlockTime * 5
+	gs := ibctypes.DefaultGenesisState()
+	gs.ClientGenesis.Params.AllowedClients = []string{"06-solomachine", "07-tendermint"}
+	gs.ConnectionGenesis.Params.MaxExpectedTimePerBlock = uint64(maxBlockTime.Nanoseconds())
```

Comment:
How come we're not returning gs?

Reply:
:see_no_evil: yikes! this was thrown together too quickly the other week, thanks for catching https://github.com/celestiaorg/celestia-app/pull/2417/commits/0d6fd942bd3b34d453a5ced73de456be9b264be2